### DISCUSSION TITLE
wz_mini_hacks, v4l2rtspserver and go2rtc

### DISCUSSION THREAD

**Discussion by johnvprcam24573455623** [blakeblackshear/frigate#7198] [General]
Since I am having many issues with Wyze cameras (camV3) I am wondering if installing the wz_mini_hacks and using v4l2rtspserver or go2rtc on the wyze and which is the best way to use it in Frigate.

As example if I enable the go2rtc server on the wyze cam then can I call it directly in the cameras:/ cameraname:  ffmpeg: without adding it in the go2rtc: - Streams 

https://github.com/gtxaspec/wz_mini_hacks/wiki/Configuration-File#go2rtc-streaming-server

**Comment by BeardedTek** [comment]
The purpose of go2rtc on wz_mini_hacks is a way to allow frigate to ingest its video.  The v4l2rtspserver implementation does not play well with frigate.  However, the go2rtc rtsp stream IS compatible with frigate.

If you use the go2rtc config, you ***should*** still use the go2rtc streams in frigate as it will serve the same function as before, limiting connections to the individual cameras which are EXTREMELY bandwidth limited since they are 2.4GHz only.
